elfi/post_proc.py

```diff
@@ -0,0 +1,106 @@
+import numpy as np
+from collections import OrderedDict
+import inspect
+
+import elfi.visualization as vis
+
+
+"""
+Implementations for post-processing.
+"""
+
+
+class Result(object):
```

Comment:
I suppose the grand idea is to have a lot of stuff under this, so Result is/will be more descriptive.